CGCharter-1727386911.html

```diff
@@ -229,7 +229,7 @@ <h2 id="decision">
     <p>
       This group will seek to make decisions where there is consensus. The Group makes decisions in the following ways: either Committers assess
       consensus, or the Chair assesses consensus, or where consensus isn't
-      clear there is a Call for Consensus [CfC] (see below) to allow multi-day
+      clear there is a <a href="#calls-for-consensus">Call for Consensus</a>  [CfC] (see below) to allow multi-day
       online feedback for a proposed course of action.
```

Comment:
This change (lines 232-233) goes along with another (lines 230-231)
```suggestion
      clear, there is a <a href="#calls-for-consensus">Call for Consensus</a>
      [CfC] (see below) to allow multi-day
      online feedback for a proposed course of action.
```